### PR TITLE
Add Wrangler CTA to the playground

### PR DESCRIPTION
**Pull request by penalosa**
## What this PR solves / how to test

Fixes https://jira.cfdata.org/browse/DEVX-1311

Adds a tab to the workers playground which calls out the C3 flow for creating a worker
Screenshot 2024-08-20 at 10 29 07

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [ ] Included
  - [x] Not necessary because: The playground frontend is manually tested
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [ ] Required / Maybe required
  - [x] Not required because: playground
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [x] Not necessary because:

### CODE REVIEW

File: packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const Link = createStyledComponent(
+	({ theme }) => ({
+		display: "inline-flex",
+		alignItems: "center",
+		width: "fit-content",
+		fontSize: theme.fontSizes[1],
+		color: `${theme.colors.indigo[isDarkMode() ? 9 : 2]} !important`,
+		backgroundColor: theme.colors.indigo[isDarkMode() ? 4 : 9],
+		borderRadius: "100vw",
+		px: 2,
```
Comment (andyjessop) [inline packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx:16]: What unit is `2`? Is it rems?

File: packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		lineHeight: 1.5,
+		verticalAlign: "bottom",
+		gap: 2,
+		"& svg": {
+			color: theme.colors.indigo[isDarkMode() ? 9 : 4],
+		},
+		"&:hover svg": {
+			color: theme.colors.indigo[isDarkMode() ? 9 : 2],
+		},
+	}),
```
Comment (andyjessop) [inline packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx:28]: I feel like these styles should be shared somehow? Is this a button that looks similar to other buttons on the site, or are these styles specific to the docs link?

File: packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { createStyledComponent } from "@cloudflare/style-container";
+import React from "react";
+
+const Link = createStyledComponent(
+	({ theme }) => ({
+		display: "inline-flex",
+		alignItems: "center",
+		width: "fit-content",
+		fontSize: theme.fontSizes[1],
+		color: `${theme.colors.indigo[isDarkMode() ? 9 : 2]} !important`,
```
[the remark this answers is not in the document]
Comment (penalosa) [reply packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx:13]: Honestly I'm not super sure—the intention here is to match the dashboard styles/code

File: packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx (hunk 1 above)
Comment (penalosa) [reply packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx:16]: No, it's a unit in our design system, which ends up rendered as `8px`

File: packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx (hunk 2 above)
Comment (penalosa) [reply packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx:28]: The styling of this button is pretty unique to the docs link—it doesn't really look like anything else on the site

**Review by andyjessop — APPROVED** [review]
Good stuff 👍

**Pull request merged**